articles/iot-edge/how-to-create-test-certificates.md

```diff
@@ -202,6 +204,9 @@ The new device identity command creates several certificate and key files, inclu
 * `<WRKDIR>\certs\iot-edge-device-identity-<name>.cert.pem`
 * `<WRKDIR>\private\iot-edge-device-identity-<name>.key.pem`
 
+For individual enrollment of the edge device in the device provisioning service (DPS), use `iot-edge-device-identity-<name>.cert.pem`, whereas for registering the iot edge device to iot hub use `iot-edge-device-identity-<name>-full-chain.cert.pem` and `iot-edge-device-identity-<name>.key.pem` certificates. For more information on how to create and provision an ioT edge device using certificates, refer [Create and provision an IoT Edge device using X.509 certificates](https://github.com/Azure/iotedge/blob/master/doc/rc/how-to-auto-provision-x509-certificates.md).
```

Comment:
* "edge device" -> "IoT Edge device"
* capitalize "Device Provisioning Service"
* capitalize "IoT Edge device"
* capitalize "IoT Hub"
* point to x.509 article in docs, not the iotedge repo: "[Create and provision an IoT Edge device using X.509 certificates](how-to-auto-provision-x509-certs.md)"